resources/nfdi4bioimage.yml

```
@@ -4483,4 +4483,19 @@ resources:
   url:
   - https://zenodo.org/records/13331351
   - https://doi.org/10.5281/zenodo.13331351
+  
+- authors: ''
```

```suggestion
- authors: 'Rémy Jean Daniel Dornier'
```

There is a box "Last edited by" which says this name. Maybe, there are more authors considering former versions....